docs/docs/Configuration/configuration-api-keys.mdx

```diff
@@ -50,6 +50,10 @@ Depending on your authentication settings, note the following requirements for c
 * If `AUTO_LOGIN` is `FALSE`, you must be logged in as a superuser.
 * If `AUTO LOGIN` is `TRUE`, you're already logged in as superuser.
 
+:::note
```

Comment:
```suggestion
:::info
```

"note" is not very semantically valuable.  However, you could also make this step 1 instead of a note.